src/sidebar/components/sidebar-panel.js

```diff
@@ -31,8 +31,11 @@ function SidebarPanel({ children, panelName, title }) {
       if (panelIsActive) {
         scrollIntoView(panelElement.current);
       }
+      if (typeof onActiveChanged === 'function') {
```

Comment:
more in favor if just `if(onActiveChanged) {...}`

it should never not be a function based on prop type. We could also pass a default to it in the prop defs? Not sure we do that much tbh?

`onActiveChanged=()=>{}`

Reply:
Any of these three approaches are reasonable. You're right that prop-type validation will ensure that `onActiveChanged` is either null/undefined or a function here. One caveat if you use the trick of having a no-op function as a default value for a prop is that a new closure will be generated every time the function is called, which can invalidate any `useEffect`/`useMemo`s etc. that depend on it.